server/startup/migrations/v082.js

```diff
@@ -0,0 +1,8 @@
+RocketChat.Migrations.add({
+	version: 82,
+	up: function() {
+		RocketChat.models.Rooms.find().forEach((room) => {
+			RocketChat.models.Rooms.incMsgCountById(room._id, RocketChat.models.Messages.find({ rid: room._id, t: { $exists: true }}).count());
```

Comment:
Actually after reviewing this, our review together is incorrect as this increments the amount of messages by how many system messages there are.